prow/cmd/deck/badge.go

```diff
@@ -85,7 +85,11 @@ func makeShield(subject, status, color string) []byte {
 
 // pickLatestJobs returns the most recent run of each job matching the selector,
 // which is comma-separated list of globs, for example "ci-ti-*,ci-other".
+// jobs will be sorted
 func pickLatestJobs(jobs []prowapi.ProwJob, selector string) []prowapi.ProwJob {
+	sort.Slice(jobs, func(i, j int) bool {
+		return jobs[i].Status.StartTime.Before(&jobs[j].Status.StartTime)
```

Comment:
Shouldn't this be After instead of Before? Otherwise the oldest job is always displayed?

Reply:
```
$ go doc sort.Slice
package sort // import "sort"

func Slice(slice interface{}, less func(i, j int) bool)
    Slice sorts the provided slice given the provided less function
```

So this does the right thing. Also, please open an issue or create a PR if you see something behaving in an unexpected way instead of commenting on age-old PRs :)